appendix-basic65.tex

```diff
@@ -2136,9 +2257,7 @@ \subsection{COLLISION}
                {\bf RSPPOS} may be used for evaluation of the sprites
                which are involved, and their positions.
 
-\item [Info:] {\bf COLLISION} wasn't completed in BASIC 10, and
-               a working implementation will be available in a future BASIC 65 update.
-
+\item [Info:] {\bf COLLISION} wasn't completed in BASIC 10. It is available in BASIC 65.
```

Comment:
My last memories of it were that it still doesn't work perfectly on our MEGA65 yet
- https://github.com/MEGA65/mega65-user-guide/issues/210

The last comment from lydon mentioned in that ticket that it has recently been fixed, so would be worth one final confirmation by running the example from the manual on the latest dev core

Reply:
I changed it because I tested the example and it worked.